tests/lib/state.sh

```diff
@@ -0,0 +1,123 @@
+#!/bin/bash
+
+SNAPD_STATE_PATH="$SPREAD_PATH"/snapd-state
+SNAPD_STATE_FILE="$SPREAD_PATH"/snapd-state.tar
+
+# shellcheck source=tests/lib/dirs.sh
+. "$TESTSLIB/dirs.sh"
+
+# shellcheck source=tests/lib/boot.sh
+. "$TESTSLIB/boot.sh"
+
+delete_snapd_state() {
+    rm -rf $SNAPD_STATE_PATH
```

Comment:
Nitpick, please quote variables. To ensure this is done please add this file to the shellcheck verification list that was recently done by @bboozzoo